doc/user/content/release-notes.md

```diff
@@ -48,6 +48,8 @@ Wrap your release notes at the 80 character mark.
 
 {{% version-header v0.9.5 %}}
 
+- Timezone parsing is now case insensitive to be compatible with PostreSQL
```

Comment:
s/PostgreSQL/PostgreSQL/

Reply:
@benesch unless my eyes are failing, these look the same to me. Did you mean something else?

Reply:
Ah, but my diff is incorrect. fixing..

Reply:
Hah, my comment must've been autocorrected. Sorry for the confusion!